xbmc/interfaces/builtins/PlayerBuiltins.cpp

```diff
@@ -812,16 +822,17 @@ static int SubtitleShiftDown(const std::vector<std::string>& params)
 ///     <b>`QueueMedia(media[\,isdir][\,1][\,playnext]\,[playoffset=xx])`</b>
 ///     \anchor Builtin_QueueMedia,
 ///     Queues the given media. This can be a playlist\, music\, or video file\, directory\,
-///     plugin or an Url. The optional parameter "\,isdir" can be used for playing
-///     a directory. "\,1" will start the media without switching to fullscreen.
-///     If media is a playlist\, you can use playoffset=xx where xx is
-///     the position to start playback from.
+///     plugin\, disc image stack\, video file stack or an URL. The optional parameter "\,isdir" can
```

Comment:
Can you use markdown inline code statements (using `) instead of the double quotes (") around isdir?

Reply:
Not introduced by me in this PR, but yes I can change this.